src/elements/common/feature-checking/FeatureFlag.js

```diff
@@ -9,16 +9,18 @@ function FeatureFlag({
     enabled = () => null,
     disabled = () => null,
     children,
+    not = false,
 }: {
     children?: React.Node,
     disabled?: () => React.Node,
     enabled?: types.FeatureOptions => React.Node,
     feature: string,
+    not: boolean,
 }) {
     return (
         <FeatureConsumer>
             {features => {
-                const isEnabled = isFeatureEnabled(features, feature);
+                const isEnabled = !not && isFeatureEnabled(features, feature);
```

Comment:
Doesn't this mean that the child will never be shown if `not` is `true`, regardless of the feature flag passed in, due to the logical AND? It seems like we'd want something similar to what's below, unless I'm missing something. Double-negatives make my head hurt:

```
const isFlagEnabled = isFeatureEnabled(features, feature);
const isEnabled = not ? !isFlagEnabled : isFlagEnabled;
```

Reply:
@jstoffan that's right - fixed and added a test case